Source/MDK-Complete.sln

```diff
@@ -1,84 +1,92 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Mdk.Extractor", "Mdk.Extractor\Mdk.Extractor.csproj", "{88AD81CF-B0C0-4D8A-B682-FC0133258810}"
```

Comment:
What's with all the changes in _this_ file?

Reply:
It is Visual Studio doing this afaik. Those are indeed not supposed to be commited (I want to keep a clean ready to test commit for me but I don't know yet how to remove some on the files for the PR candidate.